packages/launchdarkly-adapter/modules/adapter/adapter.ts

```diff
@@ -77,8 +80,15 @@ const setupFlagSubcription = ({
           [normalizedFlagName]: normalizedFlagValue,
         };
 
-        updateFlagsInAdapterState(flags);
-        onFlagsStateChange(flags);
+        const updateFlags = () => {
+          updateFlagsInAdapterState(flags);
+          onFlagsStateChange(flags);
+        };
+
+        debounce(updateFlags, {
```

Comment:
Debounce is on the trailing edge which is what we want I think. Please verify or quickly research so we ensure that.

Reply:
Verified by reading "Leading edge (or "immediate")" section in [this article](https://css-tricks.com/debouncing-throttling-explained-examples/#article-header-id-1).